src/chess/bitboard.cc

```diff
@@ -281,6 +281,19 @@ const int kKingCastleIndex =
     kMoveToIdx[BoardSquare("e1").as_int() * 64 + BoardSquare("h1").as_int()];
 const int kQueenCastleIndex =
     kMoveToIdx[BoardSquare("e1").as_int() * 64 + BoardSquare("a1").as_int()];
+
+BoardSquare Transform(BoardSquare sq, int transform) {
```

Comment:
I think it makes sense to introduce `enum` for transform values to have text representation of what every value means.

Reply:
Done.